fdbclient/NativeAPI.actor.cpp

```diff
@@ -2562,6 +2581,13 @@ ACTOR static Future<Void> tryCommit( Database cx, Reference<TransactionLogInfo>
 	if (info.debugID.present())
 		TraceEvent(interval.begin()).detail( "Parent", info.debugID.get() );
 
+	if(CLIENT_BUGGIFY) {
+		throw g_random->randomChoice(std::vector<Error>{
+				not_committed(),
+				transaction_too_old(),
+				commit_unknown_result()});
+	}
+
 	try {
```

Comment:
I'm not super familiar with how fdb client retries work, but hypothetically could the server commit without client seeing it? I.e., would it make sense to buggify in this `if` block:

https://github.com/apple/foundationdb/blob/775baf3ba004fbc07eac9870aeca815dd0a0eb37/fdbclient/NativeAPI.actor.cpp#L2619

Reply:
You're right, this should be buggified as well. Basically what can happen is that the transaction commits but the client receives a `commit_unknown_result`.